popup.js

```diff
@@ -4,6 +4,11 @@ chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
     var replacements = keywordMatching();
     registerKeywordListeners(replacements);
 	
+	if(!response.showButtons){
+		$("#allChange").hide();
+		$("#copyBtn").hide();
```

Comment:
Hiding the buttons leaves us with a whole bunch of awkward space below the rosie logo. Can we fix that, or do we not care?